rpcs3/rpcs3qt/qt_utils.cpp

```diff
@@ -625,13 +625,18 @@ namespace gui
 		{
 			usz byte_unit = 0;
 			usz divisor = 1;
+#if defined(__APPLE__)
+			constexpr multiplier = 1000; 
+#else
+			constexpr multiplier = 1024;
+#endif
 
 			static const QString s_units[]{"B", "KB", "MB", "GB", "TB", "PB"};
 
-			while (byte_unit < std::size(s_units) - 1 && size / divisor >= 1024)
+			while (byte_unit < std::size(s_units) - 1 && size / divisor >= multiplier)
```

Comment:
If we follow convention, than for decimal system the units must be in lowercase.